addons/mail/static/src/models/time/time.js

```diff
@@ -0,0 +1,47 @@
+/** @odoo-module **/
+
+import { registerModel } from '@mail/model/model_core';
+import { attr } from '@mail/model/model_field';
+
+registerModel({
+    name: 'Time',
+    identifyingFields: ['messaging'],
+    lifecycleHooks: {
+        _created() {
+            this._onEveryMinuteTimeout = this._onEveryMinuteTimeout.bind(this);
+        },
+        _willDelete() {
+            clearInterval(this.everyMinuteIntervalId);
+        },
+    },
+    recordMethods: {
+        /**
+         * @private
+         * @returns {Integer}
+         */
+        _computeEveryMinuteIntervalId() {
+            return setInterval(
+                () => this._onEveryMinuteTimeout(),
+                60 * 1000,
+            );
```

Comment:
single line, no need for arrow function as there is a bind?

Reply:
We need the arrow function, otherwise there is a lock issue:
![image](https://user-images.githubusercontent.com/1810149/148932956-a16b1e06-cd26-4c18-9413-c4cbdad47c8b.png)